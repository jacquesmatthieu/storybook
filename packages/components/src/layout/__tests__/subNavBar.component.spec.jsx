import React from 'react';
import { shallow } from 'enzyme';

import SubNavBar from '../subNavBar.component';

describe('SubNavBar', () => {
  let props;

  beforeEach(() => {
    props = {};
  });

  const getWrapper = () => shallow(<SubNavBar {...props}>Children</SubNavBar>);

  it('should render properly', () => {
    // When
    const wrapper = getWrapper();

    // Then
    expect(wrapper).toMatchSnapshot();
  });
});
