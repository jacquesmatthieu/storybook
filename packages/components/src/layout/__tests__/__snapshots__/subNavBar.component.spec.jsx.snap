// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SubNavBar should render properly 1`] = `
<subNavBarcomponent__StyledNav>
  Children
</subNavBarcomponent__StyledNav>
`;
